content/doc/developer/publishing/removing-from-distribution.adoc

```diff
@@ -0,0 +1,33 @@
+---
+title: Removing plugins from distribution
+layout: developerguide
+---
+
+In rare cases it might be necessary to remove a plugin or an individual plugin release from distribution on Jenkins project update sites.
+
+== Removing Entire Plugins
+
+Some reasons to stop distributing all versions of a plugin might be:
+
+* The plugin integrates a service with Jenkins, but the service has been shut down.
+* The plugin violates the open source license requirements of the Jenkins project.
+
+== Removing Specific Releases
+
+This is the more common case:
+A specific release of a plugin might introduce severe regressions, and fixing it is complicated and will take time.
+Or other plugins depend on the API of a given plugin, and its newest release contains accidental binary incompatible changes.
+Meanwhile, it should not be available to users.
+
+[NOTE]
+Simple fixes should just be released in a subsequent minor update.
+Reviewing requests to remove versions from distribution takes time and effort, so make sure to only do this if there's no simpler solution.
+
+[IMPORTANT]
+This will only remove versions from distribution on update sites; the corresponding artifacts will still exist in the Maven repository.
+This means, for example, that the specified version numbers will not be made available for use again.
+
+== Filing Requests
+
+To remove plugins from distribution, link:https://github.com/jenkins-infra/backend-update-center2[file a pull request for the update center generator] and include a detailed explanation for your request.
+See link:https://github.com/jenkins-infra/backend-update-center2#removing-plugins-from-distribution[that repository's README] for more details.
```

Comment:
Should we close the loop here and add a blurb clarify that requests for *actual* deletions will be rejected apart from **very** good reasons (like, leaking credz/ongoing security issue, etc.)?